_theme/_uswds-theme.scss

```diff
@@ -17,11 +16,14 @@ in the form $setting: value,
   $theme-banner-max-width: none,
   $theme-identifier-max-width: none,
 
-  // New styles for part two. Will be completed by the user.
-  $theme-card-border-color: "indigo-30",
-  $theme-card-border-radius: 3,
-  $theme-card-border-width: 1,
+  //
+  // Part two.
+  // ! Will be completed by the user.
+  //
+  // $theme-card-border-color: "indigo-30",
+  // $theme-card-border-radius: 3,
+  // $theme-card-border-width: 1,
   // TODO: Change up updating sans font family to public-sans
   // $theme-card-font-family: "serif",
-  $theme-card-header-typeset: "sans" "xl" 3
+  // $theme-card-header-typeset: 'sans' 'xl' 3
```

Comment:
Should these be double quotes to match the above definitions above this?

Reply:
Added double quotes in 12ff31a.